_sass/overrides/_main-content.scss

```diff
@@ -0,0 +1,5 @@
+.main-content a {
+  &:not(.btn) {
+    font-weight: bold;
```

Comment:
The CA State Template adds this ridiculous link underline with a background image, and we need to wipe that out. Here's what we had before, need something similar/equivalent:

```css
background-image: none !important;
background-repeat: no-repeat !important;
background-size: unset !important;
background-position: unset !important;
background-origin: unset !important;
border-bottom: solid 2px $primary;
```